lwgenerate/aws/aws.go

```diff
@@ -612,3 +636,60 @@ func createCloudtrail(args *GenerateAwsTfConfigurationArgs) (*hclwrite.Block, er
 
 	return nil, nil
 }
+
+func createAgentless(args *GenerateAwsTfConfigurationArgs) ([]*hclwrite.Block, error) {
+	if !args.Agentless {
+		return nil, nil
+	}
+
+	blocks := []*hclwrite.Block{}
+
+	integrationName := "agentless_from_terraform"
```

Comment:
I don't think we need to pass the integration name if it is not specified by the user, at
least I see a default value at https://github.com/lacework/terraform-aws-agentless-scanning/blob/main/variables.tf#L4

If we want to always customize it, let's just call it something like this:
```suggestion
	integrationName := "agentless-cli-deployment"
```
that way we will know it is coming from the CLI deployment

Reply:
Removed this to keep it simple.